lib/jxl/decode_test.cc

```diff
@@ -1782,9 +1788,12 @@ void SetPreferredColorProfileTest(
       jxl::CreateTestJXLCodestream(jxl::Bytes(pixels.data(), pixels.size()),
                                    xsize, ysize, num_channels, params);
   auto all_encodings = jxl::test::AllEncodings();
-  all_encodings.push_back(
-      {jxl::ColorSpace::kXYB, jxl::WhitePoint::kD65, jxl::Primaries::kCustom,
-       jxl::TransferFunction::kUnknown, jxl::RenderingIntent::kPerceptual});
+  // TODO(firsching): understand why XYB does not work together with icc_dst.
```

Comment:
That'd be nice to understand:)

Reply:
https://github.com/libjxl/libjxl/issues/2946